File: wrappers/dotnet/indy-sdk-dotnet/AnonCredsApi/CredentialRevokedException.cs
namespace Hyperledger.Indy.AnonCredsApi
{
    /// <summary>
    /// Exception thrown when a credential has been revoked.
    /// </summary>
    public class CredentialRevokedException : IndyException
    {
        /// <summary>
        /// Initializes a new CredentialRevokedException.
        /// </summary>
        internal CredentialRevokedException(string message) : base(message, (int)ErrorCode.AnoncredsCredentialRevoked)
        {

        }
    }

}

File: wrappers/java/src/main/java/com/evernym/vdrtools/anoncreds/CredentialRevokedException.java
package com.evernym.vdrtools.anoncreds;

import com.evernym.vdrtools.ErrorCode;
import com.evernym.vdrtools.IndyException;

/**
 * Exception thrown when attempting to use a credential that has been revoked.
 */
public class CredentialRevokedException extends IndyException
{
	private static final long serialVersionUID = 8269746965241515882L;
	private final static String message = "The credential has been revoked.";

	/**
	 * Initializes a new CredentialRevokedException.
	 */
	public CredentialRevokedException() {
		super(message, ErrorCode.AnoncredsCredentialRevoked.value());
	}
}